index.html

```diff
@@ -119,29 +119,27 @@ <h3>Concept and Initial Report</h3>
     </select>
     <div class="population-slider-container">
       <div class="population-slider-label">
-        <span class="lower value"></span>
+        <span class="population-slider-label-min"></span>
         <span> - </span>
-        <span class="upper value"></span>
+        <span class="population-slider-label-max"></span>
       </div>
-      <div class="population-slider" data-legendnum="12">
+      <div class="population-slider-controls" data-legendnum="12">
         <input
-          id="left-slider"
-          class="left-slider"
+          class="population-slider-min"
```

Comment:
Do you think `population-slider-left` is more clear? I'm leaning a bit towards it after working on some follow up PRs because we set `min` and `max for a bunch of other things like `min="0"` and `max="11"`.

Reply:
Yep! The inspiration code had `min` and `max` for slider names, but I kept getting confused, so I changed it to `left` and `right`.